### ISSUE TITLE
Fix Python virtual environment setup issue

### ISSUE THREAD

**Issue by Adityabaxi001** [iamMD01/MyGPT#1]
There is an issue with setting up the Python virtual environment when running the installation script for MyGPT. The virtual environment setup fails on systems without the python3.12-venv package installed.

Steps to Reproduce:

Run the installation script on a fresh system without the python3.12-venv package installed.
The virtual environment setup will fail.
Expected Behavior: The script should automatically install the python3.12-venv package before setting up the virtual environment.

**Comment by iamMD01** [comment]
✅ Issue Resolved!

The problem with the Python virtual environment setup has been fixed! 🎉
I've updated the installation script to automatically check for and install the python3.12-venv package if it's missing.

The script now includes this command to ensure everything runs smoothly:
sudo apt-get install -y python3.12-venv  

Thanks for flagging this! 🙌 Let me know if you encounter any other issues. 🚀
